exporter/parquetexporter/go.mod

```diff
@@ -1,51 +0,0 @@
-module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/parquetexporter
```

Comment:
We should at least deprecate the module here for one release (could be a point release as we have done previously), otherwise it will not show as deprecated on https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/exporter/parquetexporter

Reply:
Interesting, linked to https://github.com/open-telemetry/oteps/pull/232/files#r1342655110 - will do.